_includes/workshop_table.md

```diff
@@ -29,22 +29,32 @@ Click on an individual event to learn more about that event, including contact i
     </td>
 
     <td>
-        {% unless w.country == "" or w.country == "W3" %}
-      <img src="{{site.url}}/assets/img/flags/{{site.flag_size}}/{{w.country | downcase}}.png" title="{{w.country | replace: '-', ' '}}" alt="{{w.country | replace: '-', ' ' | downcase}}"  class="flags"/>
-      {% endunless %}
 
-      {% if tags contains "online" %}
+      {% if w.online == "Yes" %}
       <img src="{{site.url}}/assets/img/flags/{{site.flag_size}}/w3.png" title="Online" alt="globe image" class="flags"/>
       {% endif %}
 
-      <a href="{{w.url}}">{{ w.venue | strip_html }}{{ asterisks }}</a>
+      {% if w.in_person == "Yes" %}
```

Comment:
Consider adding a pre-test, `{% if tags contains "in_person" %}`.

Reply:
Same here, it can't be undefined with the current form